tests/use_cases/test_get_member_account.py

```diff
@@ -116,18 +117,21 @@ def test_that_correct_info_for_company_is_generated_in_correct_order_after_sever
         sending_account=company1.work_account,
         receiving_account=member.account,
         amount_received=Decimal(12),
+        date=transaction_generator.datetime_service.now() + timedelta(hours=1),
```

Comment:
Better to inject `FakeDatetimeService`. And since the `GetMemberAccount` Use Case gets transactions from the past it would be probably better if the transactions took place in the past.